tests/integration/data/interact_with_mlmd.sh

```diff
@@ -8,4 +8,15 @@ wget -O- https://github.com/fullstorydev/grpcurl/releases/download/v1.8.0/grpcur
 mkdir -p ml_metadata/proto/
 mv metadata_store.proto ml_metadata/proto/
 SERVICE=$(kubectl get services/mlmd -n $MODEL -oyaml | yq e .spec.clusterIP -)
-./grpcurl -v --proto=metadata_store_service.proto --plaintext $SERVICE:8080 ml_metadata.MetadataStoreService/GetArtifacts
+
+timeout 1m bash -x -c "
```

Comment:
From testing, I found that KubernetesServicePatch may take a few seconds to update the port, do not fail in this case.

Reply:
See https://github.com/neoaggelos/mlmd-operator/runs/3965878825?check_suite_focus=true#step:5:431 for an example run. This came up too many a time during development.